UsingSeal.md

```diff
@@ -254,49 +262,147 @@ Start by generating a master seed for the key server.
 Use the `seal-cli` tool as `cargo run --bin seal-cli gen-seed`.
 This command outputs the secret master seed which should be stored securely.
 
+```shell
+cargo run --bin seal-cli gen-seed
+Seed: $MASTER_SEED
+```
+
 Next, create a configuration file in .yaml format following the instructions in the [example config](crates/key-server/key-server-config.yaml) 
 and with the following properties:
 - Set the mode to `Permissioned`.
 - Initialize with an empty client configs (clients can be added later).
 
+```yaml
+  server_mode: !Permissioned
+    client_configs:
+```
+
 Set the environment variable `MASTER_SEED` to the master secret seed generated by the `seal-cli` tool,
 and the environment variable `CONFIG_PATH` pointing to a .yaml configuration file.
 Run the server using `cargo run --bin key-server`.
 It should abort after printing a list of unassigned derived public keys (search for logs with the text `Unassigned derived public key`).
 
+```shell
+MASTER_SEED=$MASTER_SEED CONFIG_PATH=crates/key-server/key-server-config.yaml cargo run --bin key-server 
```

Comment:
```suggestion
MASTER_SEED=$MASTER_SEED
CONFIG_PATH=crates/key-server/key-server-config.yaml
cargo run --bin key-server 
```

Reply:
same comment here, it is confusing to put this as a multiline item to run.